sql-odbc/src/sqlodbc/win_unicode.c

```diff
@@ -860,7 +860,7 @@ SQLLEN bindpara_msg_to_utf8(const char *ldt, char **wcsbuf, SQLLEN used) {
         if (used < (SQLLEN)sizeof(ntsbuf))
             ldt_nts = ntsbuf;
         else {
-            if (NULL == (alloc_nts = malloc(used + 1)))
+            if (NULL == (alloc_nts = malloc(count + 1)))
```

Comment:
does this do anything after `count = (int)used;`?

Reply:
Compiler Error:
```
/src/sqlodbc/win_unicode.c:851:9: error: variable 'count' set but not used [-Werror,-Wunused-but-set-variable]
    int count;
```
due to usages not being defined from definitions `__WCS_ISO10646__` and `__CHAR16_UTF_16__`. No functionality has changed but this way I can avoid the compiler error. See [win_unocide.c#L875-886](https://github.com/Bit-Quill/opensearch-project-sql/blob/83e059899476629425833dc2772941f2eecc8988/sql-odbc/src/sqlodbc/win_unicode.c#L875-L886)

Is there a nicer way to handle this situation?